orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/pipeline/WebhookStage.groovy

```diff
@@ -40,6 +41,10 @@ class WebhookStage implements StageDefinitionBuilder, RestartableStage, Cancella
       builder
         .withTask("monitorWebhook", MonitorWebhookTask)
     }
+    if (stage.context.containsKey("expectedArtifacts")) {
+      builder
+        .withTask(BindProducedArtifactsTask.TASK_NAME, BindProducedArtifactsTask.class);
```

Comment:
LGTM!

Java noob question: why does `BindProducedArtifactsTask.class` need to be passed here instead whereas `MonitorWebhookTask` is passed above without `.class`?

Reply:
Java (all flavors) noob answer: because copypasta. 😓 i was referencing the usage in RunJob for this.

Reply:
i can remove it to keep it consistent if need be.